### ISSUE TITLE
ParticipantsOnUpdated notifications not received 

### ISSUE THREAD

**Issue by saadali** [microsoftgraph/microsoft-graph-comms-samples#453]
If i have two participants and one bot endpoint in a call and both of the participants assigned to a TeamsComplianceRecordingPolicy, In this scenario i am unable to receive any notification on ParticipantsOnUpdated and when i check the call object participants count is 0.

Following is the Policy Attached,
![image](https://user-images.githubusercontent.com/439127/119103472-839e3900-ba1b-11eb-8ec8-4c9935851432.png)

Any help regarding this case will be appreciated.

**Comment by 1fabi0** [comment]
Does this issue occur only with ngrok or does it also happen when you have a setup without ngrok?

**Comment by saadali** [comment]
I have a setup without ngrok. The issue appears on Dev and on azure cloud service environment. Just to mention i am unable to receive notifications on endpoint bot but i am able to receive notification on recording bot.

**Comment by jsweiler** [comment]
I'll add that I've noticed the same problem as well.

**Comment by 1fabi0** [comment]
So to be clear if you manually add the bot you can't receive updates and if the policy adds your bot you receive updates?

**Comment by saadali** [comment]
No. I can explain the scenario. I have Recording Server and Call Server, if policy is assigned to only one participant in a call (means we have 4 participants, one endpoint bot, two participants and one recorder), i am receiving participant notifications on both Servers but if both caller and callee assigned with a policy (means we have 5 participants, one endpoint bot, two participants and two recorders), i am only receiving notification on Recording server not on call server.

**Comment by saadali** [comment]
> So to be clear if you manually add the bot you can't receive updates and if the policy adds your bot you receive updates?

Any update on this?
